tests/mux/test_mux.sh

```diff
@@ -118,6 +118,17 @@ rm $SOCKM $SOCKM.pid
 # check for memory leaks
 if ! [ `uname` = "FreeBSD" ]; then
 	echo "Can't check for memory leaks on `uname`"
+
+	# Kill the upstream server, and poke MUX with a new request to make it die
+	kill `cat $SOCKK.pid`
+	rm $SOCKK $SOCKK.pid
+	kill `cat $SOCKL.pid`
+	rm $SOCKL $SOCKL.pid
+	$TESTMUX $SOCKM 0. 2>/dev/null
```

Comment:
Dunno.  I copied those lines verbatim from lines 151-156 (in the modified file).  I didn't even notice that you had a different way of handling `$TEXTMUX`.  Although my guess now is that this relates to the "poke MUX with a new request to make it die" ?  Does that do some extra clean-up?